source/backend/tests/unit/dal/Repositories/InterestHolderRepositoryTest.cs

```diff
@@ -0,0 +1,204 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using FluentAssertions;
+using Moq;
+using Pims.Core.Test;
+using Pims.Dal.Entities;
+using Pims.Dal.Entities.Models;
+using Pims.Dal.Repositories;
+using Pims.Dal.Security;
+using Xunit;
+
+namespace Pims.Dal.Test.Repositories
+{
+    [Trait("category", "unit")]
+    [Trait("category", "dal")]
+    [Trait("group", "acquisition")]
+    [ExcludeFromCodeCoverage]
+    public class InterestHolderRepositoryTest
+    {
+        #region Constructors
+        public InterestHolderRepositoryTest() { }
+        #endregion
+
+        #region Tests
+
+        #region Get
+
+        [Fact]
+        public void Get_Empty()
+        {
+            // Arrange
+            var helper = new TestHelper();
+            var user = PrincipalHelper.CreateForPermission(Permissions.AcquisitionFileEdit);
+
+            var context = helper.CreatePimsContext(user, true);
+            var repository = helper.CreateRepository<InterestHolderRepository>(user);
```

Comment:
nit: these 4 lines are repeated in every test. would be inclined to move them to the test class constructor (like the controller test pattern) to avoid boilerplate